.github/workflows/nodejs.yml

```diff
@@ -27,7 +27,11 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    # - run: npm ci
-    # - run: npm run verifyMarkdown
-     # - run: npm run build --if-present
-     # - run: npm test
+    - name: npm install
```

Comment:
can we use "yarn install" instead of "npm install" command? if so then we would not need package-lock.json file in your PR